api/go.mod

```diff
@@ -0,0 +1,74 @@
+module github.com/openshift/hypershift/api
+
+go 1.17
+
+require (
+	github.com/openshift/api v0.0.0-20220124143425-d74727069f6f
+	k8s.io/api v0.23.3
+	k8s.io/apimachinery v0.23.3
+	sigs.k8s.io/cluster-api-provider-kubevirt v0.0.0-00010101000000-000000000000
```

Comment:
I would really like to drop this package, but its referenced in one of the types.

Reply:
Yeah, @csrwng also tried to do the same thing as you are doing and stumbled onto this, @rmohr wanted to look into it

Reply:
So I hacked around with copying it, but because it depends on controller-runtime (just for some silly Scheme wrapper types), we're not getting away from lots and lots of imports without rewriting the code.

Reply:
ah this one. It is a bigger undertaking that I have hoped since https://github.com/kubevirt/containerized-data-importer-api pulls in quite a bit of helper functions in from `kubevirt.io/controller-lifecycle-operator-sdk`. The task is tracked here: https://github.com/kubevirt/containerized-data-importer/issues/2029. I sadly had to refocus and bumped the issue. Thanks for the reminder.

Reply:
Ha, seems my PR was already pretty close. Managed to finish it: https://github.com/kubevirt/containerized-data-importer/pull/2207. Once this is in, we should be able to generate fresh kubevirt api packages without `controller-runtime` dependencies. Will keep you posted.

Reply:
fwiw, I don't care toooo much about controller-runtime. Better is just not pulling in all the other kubevirt deps

Reply:
Changes are slowly merging. It has to go through quite some repos and their (monthly) releases.